app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express()
+app.use(express.json())
```

Comment:
Is this code being used for this endpoint or is it boilerplate? (and therefore untested? 😬 )